index.html

```diff
@@ -16,11 +16,38 @@
 
 <body>
 
-  
+    <header>
+      <div id="divhed">
+        
+        <h1>Fun Bus</h1>
+        
+    <nav>
+      <a href="#">Home</a>
+      <a href="#">About Us</a>
+      <a href="#">Blog</a>
+      <a href="#">Contact</a>
+    </nav>
+    </div>
+    </header>
+
      
   <div class="container home">
    
+
+
+    <section class="intro-section">
+      <img class="bus-img" src="img/fun-bus.jpg">
```

Comment:
I would like to see the alt property on img tags for accessibility.
```suggestion
      <img class="bus-img" src="img/fun-bus.jpg" alt="image of a school bus">
```